services/github/github-lerna-json.service.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const Joi = require('@hapi/joi')
+const { renderVersionBadge } = require('../version')
+const { semver } = require('../validators')
+const { ConditionalGithubAuthV3Service } = require('./github-auth-service')
+const { fetchJsonFromRepo } = require('./github-common-fetch')
+const { documentation } = require('./github-helpers')
+
+const versionSchema = Joi.object({
+  version: Joi.alternatives().try(semver, Joi.equal('independent').required()),
+}).required()
+
+module.exports = class GithubLernaJson extends ConditionalGithubAuthV3Service {
+  static get category() {
+    return 'version'
+  }
+
+  static get route() {
+    return {
+      base: 'github/lerna-json/v',
+      pattern: ':user/:repo/:branch*',
+    }
+  }
+
+  static get examples() {
+    return [
+      {
+        title: 'Github lerna version',
+        pattern: ':user/:repo',
+        namedParams: { user: 'babel', repo: 'babel' },
+        staticPreview: this.render({ version: '7.6.4' }),
+        documentation,
+      },
+      {
+        title: 'Github lerna version (branch)',
+        pattern: ':user/:repo/:branch',
+        namedParams: { user: 'jneander', repo: 'jneander', branch: 'colors' },
+        staticPreview: this.render({
+          version: 'independent',
+          branch: 'colors',
+        }),
+        documentation,
+      },
+    ]
+  }
+
+  static render({ version, branch }) {
+    return renderVersionBadge({
+      version,
+      tag: branch,
+      defaultLabel: 'lerna',
+    })
+  }
+
+  async handle({ user, repo, branch }) {
+    const { version } = await fetchJsonFromRepo(this, {
+      schema: versionSchema,
+      user,
+      repo,
+      branch,
+      filename: 'lerna.json',
```

Comment:
Do we know if the `lerna.json` file must always be in the root of the repo?

Reply:
To be completely honest with you: no idea. I assumed so with the few examples I saw.

Reply:
We can probably go with this as-is, and if it's indeed possible and our users request it, we can add an optional query param to let users specify the repo path to the file

Reply:
Will give this a final look-over in a bit and then approve 👍